src/APL/fileAPL.ts

```diff
@@ -0,0 +1,85 @@
+import { promises as fsPromises } from "fs";
+
+import { APL, AuthData } from "./apl";
+
+const ENV_FILE_NAME = ".envfile";
```

Comment:
I think `.env` is a common standard, but I'd use https://www.npmjs.com/package/dotenv to manage them

Reply:
BTW this probably should be configurable in APL instance. 

I'd do `fileAPL = new FileApl(config: {envFile: string})` etc

Reply:
I'll defend the current implementation:
- it's not in the `.env` (dotenv) format - we are serializing a JSON file, so the syntax is different. To have the same syntax, I would need to use an additional package... 
- ...and `dotenv`  has support *only* for reading files. Dumping variables to the existing file would require our own serializer
- even if we had a serializer, it would mean our code would mess with `.env` file which could be created by the developer. And overriding would be tricky - we would not like to change other values (or override comments in it)
- Adding an additional package doesn't seem to be worth it in this case, where we only load/store json for this dedicated purpose

Regarding configuration - good idea